src/yardstick/tool/grype.py

```diff
@@ -83,11 +90,12 @@ def _install_from_installer(
         return Grype(path=path, version_detail=version, **kwargs)
 
     @classmethod
-    def _install_from_git(
+    def _install_from_git(  # pylint: disable=too-many-branches
```

Comment:
There are a lot of `pylint: disable` in here. Is it time to refactor?

Reply:
I've removed the need for the pylint exception with recent changes